rpc/tests/utils.rs

```diff
@@ -29,7 +30,12 @@ use kakarot_rpc_core::{client::KakarotClientImpl, utils::wiremock_utils::setup_w
 pub async fn setup_kakarot_eth_rpc() -> KakarotEthRpc {
     let starknet_rpc = setup_wiremock().await;
 
-    let kakarot_client = KakarotClientImpl::new(&starknet_rpc).unwrap();
+    let kakarot_client = KakarotClientImpl::new(
+        &starknet_rpc,
+        FieldElement::from_dec_str("0").unwrap(),
```

Comment:
either these env vars are somehow not valid, either the devnet URL inside our test suite is not working

Reply:
I used the one I removed in the tests; I think that these values are required for being consistent with the other hard coded fixtures (the json files)